pkg/cmd/server/server.go

```diff
@@ -43,11 +43,14 @@ func main() {
 		klog.Fatal("Wrong ETCD members list", etcdMembers)
 	}
 	etcdServers := strings.Split(*etcdMembers, ",")
-	db, err := ovsdb.NewDatabaseEtcd(etcdServers, *databasePrefix)
+
+	cli, err := ovsdb.NewEtcdClient(etcdServers)
```

Comment:
as the database abstraction basically implements: monitoring, transaction, locks, etcd these are mostly independent (other than sharing the client connection and agreeing on path format) so I see no reason to bundle them all together, on the contrary. So rather than the transaction fighting the database interface to to etcd operations (and use clientv3 data structures) these are all used directly.

To bridge the fact that currently code was meant to use the database abstraction is to extract the small bits of functionality which are shared namely:
- creation of etcd client
- generation of etcd key (from service, database, table, uuid)
- generation of etcd value (json serialization of row)

For now I have extract the creation of the etcd client. So you can see that the creation of the client has been made external to the database and it now get the client and does not have to create the client itself (from the endpoints)